app/helpers/appointment_helper.rb

```diff
@@ -2,4 +2,24 @@ module AppointmentHelper
   def friendly_options(statuses)
     statuses.map { |k, _| [k.titleize, k] }.to_h
   end
+
+  def rebooked_from(appointment)
+    return unless appointment.rebooked_from_id?
+
+    content_tag(
+      :span,
+      "rebooked from ##{appointment.rebooked_from_id}",
```

Comment:
I guess one scenario is quickly looking back to see when this last appointment was. 

This is making me think we could do with a 'quick jump' thing to go straight to an appointment with a given reference. Maybe we just think about this and tackle separately?